src/main/webapp/mentors.js

```diff
@@ -122,3 +122,32 @@ function createProjectsDiv(projectIDs) {
   }
   return projectsDiv;
 }
+
+function submitForm() {
+  const inputUrl = document.getElementById("inputRepo").value;
+  const encodedBody = new URLSearchParams();
+  encodedBody.append('inputRepo', inputUrl);
+  let url = new URL(
+      '/mentors', window.location.protocol + '//' + window.location.hostname);
+  const fetchRequest = fetch((url), {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: encodedBody,
+  });
+  
+  fetchRequest.then((response) => response.json()).then((response) => {
+    if (response == 'success') {
+      window.location.href = '/dashboard.html';
+    }
+    else {
+      const errorContainer = document.getElementById('error-message-container');
+      errorContainer.innerHTML = response;
```

Comment:
nit: It seems like you're only supplying text to this element. If so, it's more secure to use:
```suggestion
      errorContainer.innerText = response;
```
https://medium.com/better-programming/whats-best-innertext-vs-innerhtml-vs-textcontent-903ebc43a3fc